src/API/Site/Controllers/MerchantCenter/SupportedCountriesController.php

```diff
@@ -70,17 +72,19 @@ public function register_routes(): void {
 	 * @return callable
 	 */
 	protected function get_countries_callback(): callable {
-		return function() {
-			return $this->get_supported_countries();
+		return function( Request $request ) {
+			return $this->get_supported_countries( $request );
 		};
 	}
 
 	/**
 	 * Get the array of supported countries.
```

Comment:
We might want to update this description since it's no longer getting an array of countries.